pkg/apis/config/feature_flags_test.go

```diff
@@ -141,15 +140,15 @@ func TestNewFeatureFlagsFromConfigMap(t *testing.T) {
 				EnableAPIFields:                  "stable",
 				EmbeddedStatus:                   "full",
 				EnableCustomTasks:                config.DefaultEnableCustomTasks,
-				EnableSpire:                      true,
+				EnforceNonfalsifiability:         "spire",
 				ResourceVerificationMode:         config.DefaultResourceVerificationMode,
 				RunningInEnvWithInjectedSidecars: config.DefaultRunningInEnvWithInjectedSidecars,
 				AwaitSidecarReadiness:            config.DefaultAwaitSidecarReadiness,
 				ResultExtractionMethod:           config.DefaultResultExtractionMethod,
 				MaxResultSize:                    config.DefaultMaxResultSize,
 				CustomTaskVersion:                config.DefaultCustomTaskVersion,
 			},
-			fileName: "feature-flags-enable-spire",
+			fileName: "feature-flags-enforce-nonfalsifiability-spire",
```

Comment:
I wonder if you can also test with a bad value `ie. enforce-nonfalsifiability: 'not spire or none'`. I did that for larger results if you want to look at an example.

Reply:
Done. 

Added it in TestNewFeatureFlagsConfigMapErrors.